qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -193,13 +194,14 @@ def prep_template_post_req(study_id, user_id, prep_template, data_type,
 
     msg = ''
     status = 'success'
+    prep = False
```

Comment:
Please, define prep as None as it is done in common python practices and across the code base. This way there is no doubt if prep is a boolean or a PrepTemplate object. Then change the if to, `prep.id if prep is not None else None